src/components/Layout.js

```diff
@@ -52,6 +52,10 @@ const TemplateWrapper = ({ children }) => {
           title: "Special Events",
           path: "/programs/special-events",
         },
+        {
+          title: "Code 4Change",
```

Comment:
Is the lack of a space here intentional?

Reply:
Nope, thanks for catching it